catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/CsdidaAttribute.java

```diff
@@ -33,31 +35,120 @@
 
     private static final List<NitfAttribute<Tre>> ATTRIBUTES = new LinkedList<>();
 
-    public static final String PLATFORM_CODE = "PLATFORM_CODE";
+    static final String PLATFORM_CODE_NAME = "PLATFORM_CODE";
 
-    public static final String VEHICLE_ID = "VEHICLE_ID";
+    static final String VEHICLE_ID_NAME = "VEHICLE_ID";
 
-    public static final CsdidaAttribute PLATFORM_ID = new CsdidaAttribute(Isr.PLATFORM_ID,
+    static final String DAY_NAME = "DAY";
+
+    static final String MONTH_NAME = "MONTH";
+
+    static final String YEAR_NAME = "YEAR";
+
+    static final String PASS_NAME = "PASS";
+
+    static final String OPERATION_NAME = "OPERATION";
+
+    static final String SENSOR_ID_NAME = "SENSOR_ID";
+
+    static final String PRODUCT_ID_NAME = "PRODUCT_ID";
+
+    static final String TIME_NAME = "TIME";
+
+    static final String PROCESS_TIME_NAME = "PROCESS_TIME";
+
+    static final String SOFTWARE_VERSION_NUMBER_NAME = "SOFTWARE_VERSION_NUMBER";
+
+    private static final String ATTRIBUTE_NAME_PREFIX = "csdida.";
+
+    static final CsdidaAttribute PLATFORM_ID = new CsdidaAttribute(Isr.PLATFORM_ID,
             "PLATFORM_CODE_VEHICLE_ID",
             CsdidaAttribute::getPlatformIdFunction,
             new IsrAttributes().getAttributeDescriptor(Isr.PLATFORM_ID),
             "platformCodeVehicleId",
-            "csdida.");
+            ATTRIBUTE_NAME_PREFIX);
+
+    static final CsdidaAttribute DAY = new CsdidaAttribute("day-dataset-collection",
```

Comment:
Typically we have been following camel case convention for these NITF field names. For example, in NitfHeaderAttribute, FileProfileName is added as `ext.nitf.fileProfileName`.